GitTfs.VsCommon/TfsHelper.Common.cs

```diff
@@ -1057,9 +1055,9 @@ public IShelveset CreateShelveset(IWorkspace workspace, string shelvesetName)
             return _bridge.Wrap<WrapperForShelveset, Shelveset>(shelveset);
         }
 
-        public IIdentity GetIdentity(string username)
+        public virtual IIdentity GetIdentity(string username)
         {
-            return _bridge.Wrap<WrapperForIdentity, Identity>(Retry.Do(() => GroupSecurityService.ReadIdentity(SearchFactor.AccountName, username, QueryMembership.None)));
+            throw new NotImplementedException("You should implement this method... ;)");
         }
```

Comment:
`TfsHelperBase` is abstract. Can you change `GetIdentity()` and `GetTfsCredential()` to abstract?